graphql_compiler/compiler/ir_lowering_sql/constants.py

```diff
@@ -20,13 +20,47 @@
 
 SUPPORTED_BLOCK_TYPES = (
     blocks.QueryRoot,
+    blocks.Filter,
 )
 
 SUPPORTED_OUTPUT_EXPRESSION_TYPES = (
     expressions.OutputContextField,
 )
 
 
+class Operator(object):
+    def __init__(self, name, cardinality):
+        """Represent an operator and it's underlying method."""
+        self.name = name
+        self.cardinality = cardinality
+
+
+class Cardinality(object):
+    """Cardinality for SQLAlchemy operators."""
+
+    UNARY = 1
+    BINARY = 2
+    LIST_VALUED = 3
+
+
+SUPPORTED_OPERATORS = {
+    u'contains': Operator(u'in_', Cardinality.LIST_VALUED),
```

Comment:
Are the specific names (`Operator.name` values) here important? Are they used to look up things in SQLAlchemy? A comment might be helpful to explain why the existing compiler operator names are being remapped to new names.